src/stylesheets/base/_typography.scss

```diff
@@ -13,19 +13,19 @@
   @include text($secondary-font, normal, 1.8rem, $color-grey);
   text-align: center;
 
-  @media (min-width: 550px) {
+  @media (min-width: 34.4em) {
```

Comment:
I'd recommend not rounding so you can be pixel precise by using `34.375em` here.